examples/discord/.env.example

```diff
@@ -0,0 +1,9 @@
+DISCORD_TOKEN="REPLACEME"  # Applications > Bot > Token
+DISCORD_GUILD_ID=0  # right-click > Copy Server ID (requires Developer Mode)
+
+# See https://api.finegrain.ai/doc/
+FG_API_URL="https://api.finegrain.ai/editor"
```

Comment:
You don't need this one (it's the default).

Reply:
Ack